onboarding_project/.gitignore

```diff
@@ -0,0 +1 @@
+/squad_pantry_app/migrations/
```

Comment:
Why are migrations ignored? What happens when some one tries to run your app without migrations?

Reply:
I assumed they would need to migrate on their own. Whenever I worked with git I was told to ignore the migrations.

Reply:
Yes, they would need to run `migrate` on your own. But what does `migrate` do? What's the role of generated migration files?

Reply:
It creates the tables in DB.

Reply:
So if I want to deploy `Squad-pantry`, I will have to generate migrations on my own right? That seems redundant. So let's check in these files.

Reply:
Okay... I'll delete the .gitignore.